### ISSUE TITLE
When a node gets detached from way, the relations the node belonged to stay on the way node, rather than on the detached node

### ISSUE THREAD

**Issue by Robot8A** [openstreetmap/iD#6700]
When a node gets detached from a way, the relations the node belonged to stay on the way node, rather than on the detached node. The detached node should mantain the relations instead of the way node.

**Comment by BjornRasmussen** [comment]
I agree with this, but for some roles, such as "via", this should not be the case.

**Comment by k-yle** [comment]
This should have been auto-closed by #9816
